etc/bash-utils.sh

```diff
@@ -339,7 +339,7 @@ build-cudf-java() {
             ${D_CMAKE_ARGS} \
             -Dmaven.test.skip=true \
             -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
-            -DCUDF_CPP_BUILD_DIR="$CUDF_HOME/$CUDF_CPP_BUILD_DIR" \
+            -DCUDF_CPP_BUILD_DIR="$CUDF_CPP_BUILD_DIR" \
```

Comment:
Does this need to be a relative path? If absolute is OK, this could be
```
-DCUDF_CPP_BUILD_DIR="$CUDF_ROOT"
```

Reply:
I don't know, but it needs to evaluate to `$CUDF_ROOT/cpp/build/release`, which is what the function that sets `CUDF_CPP_BUILD_DIR` finds, I think.

Reply:
`$CUDF_ROOT` will be the path to the current build dir:
```
$ echo $CUDF_ROOT
/home/ptaylor/dev/rapids/cudf/cpp/build/release
```

But this change looks OK too.

Reply:
I did not know that!